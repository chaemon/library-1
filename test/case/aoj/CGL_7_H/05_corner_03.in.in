9 3
0 -1
-1 0
-4 -4
-4 -6
4 -6
4 1
2 1
2 -5
-3 -5
