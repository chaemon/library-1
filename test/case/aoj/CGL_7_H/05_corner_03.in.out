5.6713918108
